README.md

```diff
@@ -74,4 +74,9 @@ So for each lib we will create our own interpreter. Also add env variables for e
 
 For example for lib `php-table-backend-utils` as seen in the picture.
 
-![Cli interpreter in phpstorm](cli-interpreter-table-utils.png?raw=true "Cli interpreter in phpstorm")
\ No newline at end of file
+![Cli interpreter in phpstorm](cli-interpreter-table-utils.png?raw=true "Cli interpreter in phpstorm")
+
+1. Then you need to configure Test Framework in `PHP->Test Frameworks`
+2. Click on + and select PHPUnit by Remote Interpreter ![PHPUnit by Remote Interpreter](add-phpunit-local.png?raw=true "PHPUnit by Remote Interpreter")
+3. And in path to script select the path to `keboola/storage-backend/vendor/autoload.php` ![Path to autoload](path-to-autload.png?raw=true "Path to autoload")
+4. Click ok
```

Comment:
:thinking: to by šlo nascriptovat pro celé monorepo stejně jak to dělám u našeho generátoru projektů v connection

Reply:
Ok hodim si na to task

Reply:
to je jen myšlenka :) ale klidně. Tady stím manipuluju v generátoru https://github.com/keboola/connection/blob/master/provisioning/projects/src/Output/PhpstormOutputWriterAdapter.php

Reply:
jj, len aby som na to proste nezabudol.